reactor-netty-http-brave/src/test/java/reactor/netty/http/client/ITBraveHttpClientTracingTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2011-Present VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.client;
+
+import brave.test.http.ITHttpClient;
+import io.netty.handler.codec.http.HttpMethod;
+import reactor.core.publisher.Mono;
+import reactor.netty.ConnectionObserver;
+
+import java.time.Duration;
+
+public class ITBraveHttpClientTracingTest extends ITHttpClient<HttpClient> {
+
+	@Override
+	protected HttpClient newClient(int port) {
+		ConnectionObserver observer = BraveHttpClientTracing.create(httpTracing, s -> null);
+
+		return HttpClient.create()
+		                 .host("127.0.0.1")
+		                 .port(port)
+		                 .wiretap(true)
+		                 .followRedirect(true)
+		                 .disableRetry(true)
+		                 .observe(observer);
+	}
+
```

Comment:
add an extra test that shows a user accessing the SpanContext

ex https://github.com/openzipkin/brave/blob/master/instrumentation/httpasyncclient/src/test/java/brave/httpasyncclient/ITTracingHttpAsyncClientBuilder.java#L79-L95